src/encoded/audit/experiment.py

```diff
@@ -1136,10 +1137,18 @@ def check_experiment_chip_seq_standards(
         reps=experiment['replicates'],
         excluded_types=[]
     )  # human/mouse
+
+    target = get_target(experiment)
+    if target is False and not experiment.get('control_type'):
+        return
+
+    # Handles library_complexity for all cases and read_depth for all but H3K9me3
     for f in alignment_files:
-        target = get_target(experiment)
-        if target is False and not experiment.get('control_type'):
-            return
+
+        yield from check_file_chip_seq_library_complexity(f)
+
+        if target and target['name'] in ['H3K9me3-human', 'H3K9me3-mouse']:
+            break
 
```

Comment:
I think that the original behavior is altered after this code. For instance in H3K9me3 only one "filtered alignment" file will be used to check the library complexity, instead of all of the "filtered alignments"

Reply:
Agreed, swapped 'continue' in for 'break'